libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -975,6 +983,11 @@ AP_GPS_UBLOX::_parse_gps(void)
                     (unsigned)_buffer.gnss.configBlock[i].flags);
                 }
 #endif
+                if (_buffer.gnss.configBlock[1].flags & 0x1) {
+                    GCS_SEND_TEXT(MAV_SEVERITY_ALERT, "SBAS Enabled!\n");
+                } else {
```

Comment:
Are these going to be noisy alerts?

Reply:
It's possible, yes. We've encountered it in the past during Likely Failure testing, but it wasn't noisy enough to be a hindrance.

Reply:
Okay, sounds good.